commands/volumes/volume-create.go

```diff
@@ -120,6 +121,39 @@ func rollBackVolumeCreate(c *context.Context) error {
 	return nil
 }
 
+func registerVolCreateStepFuncs() {
+	var sfs = []struct {
+		name string
+		sf   transaction.StepFunc
+	}{
+		{"vol-create.Stage", validateVolumeCreate},
+		{"vol-create.Commit", generateVolfiles},
+		{"vol-create.Store", storeVolume},
+		{"vol-create.Rollback", rollBackVolumeCreate},
```

Comment:
This function only removes brick paths. Should gluster do it ?

Besides, I don't see any code to remove volfiles generated in generateVolfiles.